components/src/core/dot-stepper/dot-stepper.component.scss

```diff
@@ -0,0 +1,17 @@
+.pxb-dot-stepper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.pxb-dot-stepper-dot {
+    height: 8px;
+    width: 8px;
+    background-color: gray;
```

Comment:
Why is this color style here? Do they belong to pxb themes instead?

Reply:
I've added in temporary styles because otherwise this component will not render anything.  I think it would be a good idea to have some sort of default colors in the event someone is not using our themes and expects to visually see something.